unlock-protocol-com/next.config.js

```diff
@@ -59,19 +59,4 @@ module.exports = {
     })
     return config
   },
-  exportPathMap: async (defaultPathMap, { dir }) => {
```

Comment:
Don't we need that to make sure the static pages are built ahead of time?

Reply:
Nope. Next js generates a static page automatically based on the lage and posts where we need external data at build time, you can use getStaticProps function which I have done. 

https://nextjs.org/docs/advanced-features/automatic-static-optimization

https://nextjs.org/docs/basic-features/data-fetching/get-static-props